beacon-chain/p2p/peers/scorers/bad_responses.go

```diff
@@ -58,6 +58,9 @@ func (s *BadResponsesScorer) Score(pid peer.ID) float64 {
 
 // score is a lock-free version of Score.
 func (s *BadResponsesScorer) score(pid peer.ID) float64 {
+	if s.isBadPeer(pid) {
+		return BadPeerScore
```

Comment:
in that case, you only return `-1` . Isn't that very low for a 'bad' peer ?

Reply:
All the scores will be within [0..1] range (signed). Then you can change scorer weight, and turn that into anything you want (for example bad responses scorer can have a magnifying coefficient x10, and turn that into -10).